src/components/LogoutLink.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'react-router/lib/Link';
+
+/**
+ * Logout link component
+ * @module LogoutLink
+ */
+class LogoutLink extends React.Component {
+	render() {
+		const {
+			to,
+			children,
+			...other
+		} = this.props;
+
+		return (
+			<Link
+				to={{ pathname: to, query: {logout: true} }}
```

Comment:
Currently, any sort of routing navigation drops all query params. i.e., `localhost:8000/?logout` -> click `Login` -> `localhost:8000/login`. However, if you got to `localhost:8000/login?logout` -> submit login -> you get an invalid cookie error.

```
Error: Invalid cookie value
Items.serial
/Users/sadaf/git_repo/mup-web/node_modules/hapi/node_modules/statehood/lib/index.js:244:43
done
/Users/sadaf/git_repo/mup-web/node_modules/hapi/node_modules/items/lib/index.js:31:25
internals.decode
/Users/sadaf/git_repo/mup-web/node_modules/hapi/node_modules/statehood/lib/index.js:201:32
Iron.unseal
/Users/sadaf/git_repo/mup-web/node_modules/hapi/node_modules/statehood/lib/index.js:311:24
process.nextTick
/Users/sadaf/git_repo/mup-web/node_modules/hapi/node_modules/hoek/lib/index.js:854:22
_combinedTickCallback
internal/process/next_tick.js:67:7
process._tickDomainCallback
internal/process/next_tick.js:122:9
```

i'll file a defect